docs/conf.py

```diff
@@ -57,3 +57,4 @@
 
 # Configuration for https://autodoc-pydantic.readthedocs.io
 autodoc_pydantic_model_show_config_summary = False
+autodoc_pydantic_field_show_alias = False
```

Comment:
We use Pydantic aliases purely so we can deserialize camelCase to snake_case and vice versa, so there's no reason to display them in the docs.